docs/usage/self-hosted-configuration.md

```diff
@@ -330,6 +330,10 @@ To create the key pair with OpenSSL use the following commands:
 - `openssl genrsa -out rsa_priv.pem 4096` for generating the private key
 - `openssl rsa -pubout -in rsa_priv.pem -out rsa_pub.pem` for extracting the public key
 
+To encrypt a secret using openssl use following command:
+
+- `echo 'your-secret' | openssl rsautl -encrypt -pubin -inkey rsa_pub.pem | base64`, replacing `your-secret` with your actual secret
+
```

Comment:
```bash
echo 'your-secret' | openssl rsautl -encrypt -pubin -inkey rsa_pub.pem | base64
```

Replace `your-secret` with your actual secret.

---

Please make it look like the section above.
I don't know how to suggest this in a way that GitHub will accept unfortunately.

Reply:
Thanks, added it. 
> Please make it look like the section above.
Or do you mean I should add it just as a bullet point to the two commands above?

Reply:
Thank you, but I wasn't clear enough on the change I wanted you to make... 😉 

Please put the code in a code block, like this:

```bash
echo 'your-secret' | openssl rsautl -encrypt -pubin -inkey rsa_pub.pem | base64
```

To do this: 

1. Remove bullet point `-` character.
2. Wrap your code into a code block (3 backticks, code, 3 backticks).
3. Put the comment about what to replace below the codeblock.

https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Reply:
Got you now, thanks for the clarification. 
Incorporated it now 👍

Reply:
We got there in the end! 😄 🎉 ✨